ee/vellum_cli/push.py

```diff
@@ -229,6 +228,12 @@ def push_command(
         raise e
 
     if dry_run:
+        workflow = BaseWorkflow.load_from_module(workflow_config.module)
+        workflow_display = get_workflow_display(
+            workflow_class=workflow,
+            client=client,
+            dry_run=dry_run or False,
+        )
```

Comment:
Damn ok - update the return type of `load_from_module` to be a new pydantic model that looks like:
- `exec_config: dict`
- `errors: list[Exception]`

so now we don't need to load from module again here